### PR TITLE
fix: Fixed a problem with misspellings and markdown syntax errors

### PR DESCRIPTION
**Pull request by creanme**
## PR Checklist

- [x] Fix linting errors
- [x] Label has been added

## Change information

Fixed a problem with misspellings and markdown syntax errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -101,21 +101,21 @@
...
+    ex:
+    ```
+    const MAX = '100', datePipe = (date) => { ... }
+    // ...
+    // in your component
+
+    <div>{{ max }} {{datePipe('2018/12/01')}}</div>
+
+    @Decorators.Shortcut({
+    max: MAX, datePipe: datePipe,
```
Comment (creanme) [inline README.md]: 缩进弄没了

File: README.md [hunk 2]
```diff
@@ -101,21 +101,21 @@
...
+    })
+    data: () => ({
+    your responsive datas...
```
Comment (creanme) [inline README.md]: 缩进弄没了

File: README_CN.md [hunk 3]
```diff
@@ -101,21 +101,21 @@
...
+    // ...
+    // 在组件中使用 Shortcut 绑定数据
+
+    <div>{{ max }} {{datePipe('2018/12/01')}}</div>
+
+    @Decorators.Shortcut({
+    max: MAX, datePipe: datePipe,
+    })
+    data: () => ({
+    your responsive datas...
```
Comment (creanme) [inline README_CN.md]: 缩进弄没了

Comment (creanme) [inline README_CN.md]: 缩进弄没了

**Comment by WittBulter** [comment]
怎么 close 了？

**Comment by creanme** [comment]
> 怎么 close 了？

CI检查通不过啊

**Pull request closed without merging**